rails_translation_manager.gemspec

```diff
@@ -25,4 +25,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest"
   spec.add_development_dependency 'rspec'
+  spec.add_development_dependency 'pry'
```

Comment:
Could this be moved into a separate commit, as it's unrelated to this change? But I agree it's worth adding - though you may want to consider switching to [byebug](https://rubygems.org/gems/byebug) instead, which is the one we seem to be switching to.

Reply:
@ChrisBAshton Have added byebug in separate commit.